docs/pages/web/sheet.js

```diff
@@ -141,7 +142,18 @@ When Sheet opens, focus should be placed on the first interactive element within
       </AccessibilitySection>
       <MainSection
         name="Localization"
-        description={`Be sure to localize the \`heading\`, \`accessibilityDismissButtonLabel\` and \`accessibilitySheetLabel\` props. Note that localization can lengthen text by 20 to 30 percent.`}
+        description={`Be sure to localize the \`heading\`, \`accessibilityDismissButtonLabel\`, \`accessibilitySheetLabel\` props and well as any custom string in \`dismissConfirmation\`. Note that localization can lengthen text by 20 to 30 percent.`}
```

Comment:
typo: "…custom _strings_ in `dismissConfirmation`."

Reply:
DONE